audio/src/main/res/layout/view_number_pad_delete_back_element.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- TODO: Consolidate with view_number_pad_numeric_element.xml -->
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/number_frame"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_margin="@dimen/marginStandard"
+    tools:context=".AudioActivity"
+    tools:layout_gravity="center"
+    >
+
+    <TextView
+        android:id="@+id/number_content"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:layout_margin="1dp"
+        android:background="@drawable/number_pad_delete"
+        android:gravity="center"
+        android:includeFontPadding="false"
+        android:maxLength="1"
+        android:textAlignment="center"
+        android:textColor="@color/white"
+        android:textSize="26sp"
```

Comment:
Same here, extract the size value to `dimens`.

Reply:
utilize an existing zeplin style (or add it). Make sure its there otherwise talk to design.